migrations/00277_optimize_mediaitems_view.sql

```diff
@@ -0,0 +1,106 @@
+-- +goose Up
+
+-- +goose StatementBegin
+CREATE OR REPLACE FUNCTION mediaitems_by_episodes(episodeids int[])
+    RETURNS TABLE
```

Comment:
maybe try OUT params, since this PR is mentioning that: https://github.com/sqlc-dev/sqlc/pull/3175

Reply:
Unfortunately you can't combine them: `[42804] ERROR: RETURN cannot have a parameter in function with OUT parameters`

Reply:
The other alternative would be to create a materialized view for the innermost SELECTS, then we could probably get away with a "normal" view.

Reply:
Now using "setof" option for returning from a function
This takes care both of needing a real DB connection, as well as nullable fields.